### ISSUE TITLE
(Chat Item not Issue)

### ISSUE THREAD

**Issue by GaryFurash** [mgsloan/todoist-shortcuts#253]
Is there any way we can get Todoist to incorporate some of the shortcuts here into their product?

**Comment by champagnealexandre** [comment]
I wouldn’t count on it. Todoist is a nice app (the only one I’d use besides Omnifocus) but they don’t listen to user suggestions much. And app development is moving quite slow.

So it will likely never happen, happen much farther in the future, not be implemented the way we like it or get broken eventually. Better stick with jscript/CSS.

**Comment by adamleerich** [comment]
I've been using Todoist and @mgsloan's add-in for several years now, and the Todoist people have very much broadened their built-in shortcut keys.  To the extent that has been because of this add-in, I do not know.  But, I know that Michael has said in a couple of different places that the Todoist team know about and support this work.  So, @GaryFurash, I think it is fair to say that Todoist are incorporating more shortcuts slowly, but whether they will build in all provided by Michael's add-in the answer is probably "no."

**Comment by mgsloan** [comment]
Yeah, I believe many of Todoist's builtin shortcuts were inspired by the extension, which is great.

It would be nice to have less maintenance.  Right now changes in Todoist sometimes break the extension.  One approach that might be taken in the future might be to have the extension reuse the builtin keybindings (and remap when todoist-shortcut's shortcuts differ).

This would greatly reduce the amount of code in the extension and make it less fragile.  Some large chunks would still remain - I imagine they probably won't add anything like the `g` navigation mode as that is a fairly unorthodox UI, so that would still be in the extension.
